archive-tools/do-release.pl

```diff
@@ -14,9 +14,41 @@
 my $homedir = glob("~openssl");
 my $tmpdir  = $ENV{"OPENSSL_TMP_DIR"} // $homedir . "/dist/new";
 my $olddir  = $ENV{"OPENSSL_OLD_DIR"} // $homedir . "/dist/old";
-my $ftpdir  = $ENV{"OPENSSL_FTP_DIR"} // "/srv/ftp/source";
 my $mail    = $ENV{"OPENSSL_MAIL"} // "mutt -s SUBJECT RECIP < BODY";
```

Comment:
The mutt command needs updating to match what it says here:

https://github.com/openssl/tools/blob/c60e4aa252e6b1a76b097c4b4ca2e427f79465ef/HOWTO-make-a-release.md?plain=1#L356-L358

The HOWTO also says "They should be sent from the account of the person that owns the key used for signing the release announcement." - so sending them direct from this script (which I never do), is slightly problematic since it will be sent from the account of the user running the script (probably "openssl"), which is probably not the same as the account of the person that signed the announcement. This is important because the "from" address and PGP key address need to match - otherwise some PGP clients show a bad signature.

Reply:
Done in a separate commit.
[aaaand back to vacationing...]